helm/cas-ciip-portal/Chart.yaml

```diff
@@ -12,3 +12,19 @@ dependencies:
     version: "3.3.0"
     repository: "https://codecentric.github.io/helm-charts/"
     condition: mailhog.enable
+  - name: cas-airflow-dag-trigger
+    version: 1.0.5
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-airflow-ciip-cert-renewal
+  - name: cas-airflow-dag-trigger
+    version: 1.0.5
+    repository: https://bcgov.github.io/cas-airflow
+    alias: download-airflow-ciip-deploy-db
```

Comment:
Ideally we'd have only one "download" DAG, either by updating it to support downloading a directory, or by having a single DAG file